src/main/java/org/broadinstitute/consent/http/service/SupportRequestService.java

```diff
@@ -32,6 +34,8 @@ public class SupportRequestService {
     private final ServicesConfiguration configuration;
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
+    private static final String TASK_REQUEST_TYPE = "task";
```

Comment:
Would be better to create a `RequestType` enum for the `createSupportTicket` to take as an argument instead of `String type`? This is the Zendesk documentation on type: `The type of the request, "question", "incident", "problem", "task"`. If so, I can create a separate PR for this ticket